app/(landing)/blog/tag/[slug]/page.tsx

```diff
@@ -0,0 +1,64 @@
+import NotFoundPage from "@/app/not-found";
+import { PostItem } from "@/app/types";
+import BlogItem from "@/components/blog/BlogItem";
+import BlogPlatform from "@/components/blog/BlogPlatform";
+import Breadcrumb from "@/components/breadcrumb/Breadcrumb";
+import { GET_POSTS_BY_TAGS_QUERY } from "@/graphql/queries/post.query";
+import { isrClient } from "@/lib/apolloClient";
+
+interface Props {
+  params: {
+    slug: string;
+  };
+}
+
+const BlogTag = async({params}: Props) => {
+
+  const postsByTagID = await isrClient.query({
+    query : GET_POSTS_BY_TAGS_QUERY,
+    variables: { tag: params.slug, first: 10, author: 3 },
```

Comment:
the name `GET_POSTS_BY_TAGS_QUERY` is not good. We only query only one tag, not many tags.